src/main/java/io/jawg/osmcontributor/ui/activities/PhotoActivity.java

```diff
@@ -377,7 +385,7 @@ private void uploadPhoto() {
         flickrUploadClient.upload(typedFile, new Callback<String>() {
             @Override
             public void success(String s, Response response) {
-                onUploadFinishedEVent(ResponseConverter.convertImageId(s));
+                onUploadFinishedEvent(ResponseConverter.convertImageId(s));
```

Comment:
As this one seems to be moving to a "pipeline"-like process (with subsequent async steps), would you consider using RxJava2 for next time?
It would probably clean up the callback hell and help with removing spaghetti code ^^
This should have been made a long time ago though :(